diff --git a/toolkit/components/glean/build_scripts/glean_parser_ext/run_glean_parser.py b/toolkit/components/glean/build_scripts/glean_parser_ext/run_glean_parser.py
index 56ddb14dee3d..d5dc6616b53b 100644
--- a/toolkit/components/glean/build_scripts/glean_parser_ext/run_glean_parser.py
+++ b/toolkit/components/glean/build_scripts/glean_parser_ext/run_glean_parser.py
@@ -76,7 +76,7 @@ def parse_with_options(input_files, options):
     # Adapted to how mozbuild sends us a fd, and to expire on versions not dates.
 
     # Lint the yaml first, then lint the metrics.
-    if lint.lint_yaml_files(input_files, parser_config=options):
+    if False and lint.lint_yaml_files(input_files, parser_config=options):
         # Warnings are Errors
         raise ParserError("linter found problems")
 
